project/LinkerdBuild.scala

```diff
@@ -69,7 +69,7 @@ object LinkerdBuild extends Base {
   }
 
   val configCore = projectDir("config")
-    .withTwitterLibs(Deps.finagle("core"), Deps.finagle("thrift"))
+    .withTwitterLibs(Deps.finagle("core"))
```

Comment:
`ThriftProtocolDeserializer` imports `import com.twitter.finagle.thrift.Protocols`  
isn't `finagle-thrift` needed for that?